src/components/Modal/README.md

```diff
@@ -346,8 +346,8 @@ class ModalExample extends React.Component {
           onClose={this.toggleModal}
           title="Get a shareable link"
           primaryAction={{
-            content: 'Close',
-            onAction: this.toggleModal,
+            content: 'Copy link',
```

Comment:
Why this change? I would expect a modal dialog to either have a confirmation action paired with cancel, or a simple close button like in the original example. I would expect any of these actions to close the modal, and I don’t feel like the copy link action fits that expectation.